pkg/apis/operator/v1alpha1/knativeserving_types.go

```diff
@@ -112,6 +116,14 @@ type IngressConfigs struct {
 // IstioIngressConfiguration specifies options for the istio ingresses.
 type IstioIngressConfiguration struct {
 	Enabled bool `json:"enabled"`
+
+	// KnativeIngressGateway overrides the knative-local-gateway.
```

Comment:
s/knative-local-gateway/knative-ingress-gateway

Reply:
Thank you. Done.